editions/tw5.com/tiddlers/macros/TagMacro.tid

```diff
@@ -11,7 +11,33 @@ The <<.def tag>> [[macro|Macros]] generates a tag pill for a specified tag. Clic
 
 !! Parameters
 
-;tag
+; tag
 : The title of the tag, defaulting to the [[current tiddler|Current Tiddler]]
 
+!! CSS classes
+
+<<.from-version "v5.3.4">>
+
+; `tc-tag-missing`
+: This class is defined if a tag does ''not exist'' as a tiddler. By default the format is set to //italic//
```

Comment:
How is the default italic text if there are no default CSS definitions for it in the core?

Reply:
uups. forgot to change that